product_docs/docs/pgd/5/known_issues.mdx

```diff
@@ -2,56 +2,50 @@
 title: 'Known issues'
 ---
 
-This section discusses currently known issues in EDB Postgres Distributed 5. 
-These known issues are tracked in PGD's
-ticketing system and are expected to be resolved in a future
-release.
-
--   If the resolver for the `update_origin_change` conflict
-    is set to `skip`, `synchronous_commit=remote_apply` is used, and
-    concurrent updates of the same row are repeatedly applied on two
-    different nodes, then one of the update statements might hang due
-    to a deadlock with the PGD writer.  As mentioned in the
-    [Conflicts](consistency/conflicts/) chapter, `skip` is not the default
-    resolver for the `update_origin_change` conflict, and this
-    combination isn't intended to be used in production. It discards
-    one of the two conflicting updates based on the order of arrival
-    on that node, which is likely to cause a divergent cluster.
-    In the rare situation that you do choose to use the `skip`
-    conflict resolver,  note the issue with the use of the
-    `remote_apply` mode.
-
--   The Decoding Worker feature doesn't work with CAMO/EAGER/Group Commit. Installations using CAMO/Eager/Group Commit must keep `enable_wal_decoder` disabled.
+This section discusses currently known issues in EDB Postgres Distributed 5.
+These known issues are tracked in PGD's ticketing system and are expected to be
+resolved in a future release.
+
+-   The Decoding Worker feature doesn't work with CAMO/EAGER/Group Commit.
+    Installations using CAMO/Eager/Group Commit must keep `enable_wal_decoder`
+    disabled.
```

Comment:
So far, this "known issues" list has been generic and potentially spanning multiple major releases.

Speaking of "known issues in PGD 5" above and listing "decoding worker vs CAMO/Eager/Group Commit" here could be interpreted as a promise to "fix" this within the PGD 5 major version. I don't think we can promise that, though.

Reply:
We've historically mentioned the major version at the top of the known issues. Going forward we should consider where something is resolved in a minor version, marking the issue resolved as (Versions before 5.1) which would also make reasons to upgrade clearer.